lib/players/human.rb

```diff
@@ -0,0 +1,13 @@
+module Players
+  class Human < Player
+
+    def move(board)
+      puts ""
+      puts "Player #{@token}, make your move! [1-9]"
+      input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```ruby
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```